testing/unit_testing/count_cohorts_test/test_CountCohorts.pf

```diff
@@ -0,0 +1,98 @@
+module test_CountCohorts
+  !
+  ! DESCRIPTION:
+  !		Tests the patch's CountCohorts method
+  !
+  use FatesConstantsMod, only : r8 => fates_r8
+  use FatesCohortMod,    only : fates_cohort_type
+  use FatesPatchMod,     only : fates_patch_type
+  use funit
+
+  implicit none
+  
+  @TestCase
+  type, extends(TestCase) :: TestCountCohorts
+    type(fates_patch_type), pointer :: patch
```

Comment:
Is there a reason the patch needs to be a pointer here? (Maybe there really is, in which case you can ignore this question!)

Reply:
no... I think I just put that there out of habit haha (the production-code patches actually ARE pointers). thanks for catching that!